content/en/security/application_security/setup/java/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Enabling AAP for Java
+code_lang: java
+type: multi-code-lang
+code_lang_weight: 0
+aliases:
+  - /security_platform/application_security/getting_started/java
+  - /security/application_security/getting_started/java
+  - /security/application_security/threats/setup/threat_detection/java
+  - /security/application_security/threats_detection/java
+  - /security/application_security/setup/aws/fargate/java
+further_reading:
+- link: "/security/application_security/add-user-info/"
+  tag: "Documentation"
+  text: "Adding user information to traces"
+- link: 'https://github.com/DataDog/dd-trace-java'
+  tag: "Source Code"
+  text: 'Java Datadog library source code'
+- link: "/security/default_rules/?category=cat-application-security"
+  tag: "Documentation"
+  text: "OOTB App and API Protection Rules"
+- link: "/security/application_security/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshooting App and API Protection"
+---
+{{< partial name="api_security/callout.html" >}}
+
+{{< partial name="api_security/java/overview.html" >}}
+
+## Environments
+
+{{< appsec-integrations >}}
+  {{< appsec-integration name="Docker" avatar="docker" link="./docker" >}}
+  {{< appsec-integration name="Kubernetes" avatar="kubernetes" link="./kubernetes" >}}
+  {{< appsec-integration name="Linux" avatar="linux" link="./linux" >}}
+  {{< appsec-integration name="macOS" avatar="apple" link="./macos" >}}
+  {{< appsec-integration name="Windows" avatar="windows" link="./windows" >}}
+{{< /appsec-integrations >}}
```

Comment:
can you keep the same grouping as the main entry page please?

- Hosts
  - Windows
  - Linux
  - MacOS
- Cloud and Container Platforms
  - Kubernetes